benchmarks/cheshire/test/benchmark.clj

```diff
@@ -51,6 +56,13 @@
     (bench/bench (core/decode (core/encode test-obj)) :verbose))
   (println "-------------------------------------"))
 
+(deftest t-bench-pretty
+  (println "------- PrettyPrint Benchmarks ------")
+  (println "........default pretty printer")
+  (bench/bench (core/encode test-obj {:pretty true}))
+  (println "........custom pretty printer")
+  (bench/bench (core/encode test-obj test-pretty-opts)))
```

Comment:
So that the pretty printer doesn't have to be created every time, perhaps this could be:

```clj
(let [printer (create-pretty-printer test-pretty-opts)]
  (bench/bench (core/encode test-obj printer)))
```

What do you think?